src/test/java/com/google/javascript/clutz/ProgramSubject.java

```diff
@@ -96,7 +96,7 @@ void generatesDeclarations(File golden) throws IOException {
           && expected.equals(expectedClean)) {
         Files.write(stripped, golden, Charsets.UTF_8);
       } else {
-        failComparing("compilation result doesn't match", expected, stripped);
```

Comment:
failComparing makes it show a diff, does the new thing?

I think I made a similar change elsewhere, where I found that failComparing had just moved locations in the API

Reply:
It shows as:

```
com.google.javascript.clutz.DeclarationGeneratorTests > colliding_provide.js FAILED
    expected:<...rt default alias;
    }
    [// some new text]> but was:<...rt default alias;
    }
    []>
```

and I still get my see diff link in Intellij.

Reply:
Tested old emit for the same error:
```
    compilation result doesn't match expected:<...rt default alias;
    }
    [// some new text]> but was:<...rt default alias;
    }
    []>
``` 
basically the same just text has changed.

Reply:
Can you make the CL more descriptive? e.g. "failComparing is deprecated" or some such?

Reply:
Also the internal CL says we need to change truth versions for this, have we done that?

Reply:
Upgraded to Truth 0.40 (thought it didn't seem needed for the change). PTAL